infra/k8s/sidecar.yaml

```diff
@@ -30,13 +31,13 @@ spec:
           privileged: true
         env:
         - name: REDIS_HOST
-          value: "redis-master"
+          value: "redis-headless"
```

Comment:
This change still assumes that we are going to resolve `redis-headless` only once, and cache the IP. Once we drop the `default` route, we won't be able to resolve it again, until we fix https://github.com/ipfs/testground/issues/390